plugin/metrics/prometheus/options.go

```diff
@@ -21,14 +21,15 @@ import (
 
 	"github.com/spf13/viper"
 
+	"github.com/jaegertracing/jaeger/pkg/config/tlscfg"
 	"github.com/jaegertracing/jaeger/pkg/prometheus/config"
 )
 
 const (
-	suffixHostPort       = ".host-port"
+	suffixServerURL      = ".server-url"
```

Comment:
Motivation for renaming:

- To use TLS with Prometheus requires the protocol scheme `https`, as with how ES's server URLs are defined. Hence, I felt it made more sense that users are required to provide the appropriate protocol scheme of either `http` or `https`.
- Consistency with ES options `server-urls`, and made more sense than `host-port` if requiring the protocol scheme.